.gitignore

```diff
@@ -31,3 +31,7 @@ status-react-translations/
 /.update.timestamp
 notarization.log
 status-desktop.log
+<<<<<<< HEAD
```

Comment:
left over?

Reply:
@Khushboo-dev-cpp you're right, I missed it somehow, most likely cause it's in hidden files or whatever, anyway it's removed.